wasted-app/Model/Repositories/BaseRepository.cs

```diff
@@ -11,39 +11,39 @@ public abstract class BaseRepository<T> : IBaseRepository<T> where T : BaseEntit
     {
         private readonly string _pathToDatabase;
 
-        public BaseRepository(String pathToDatabase)
+        public BaseRepository(string pathToDatabase)
         {
             _pathToDatabase = pathToDatabase;
         }
 
         public void Add(T entity)
         {
-            List<T> all = GetAll().ToList();
+            var all = GetAll().ToList();
```

Comment:
was backend code supposed to be refactored to vars only?

Reply:
yeah I was wondering that too